<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QWidget" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图像几何变换</string>
  </property>
  <widget class="QPushButton" name="openImgBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>打开图片</string>
   </property>
  </widget>
  <widget class="QLabel" name="oriImgLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="edgeImgLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edgeImgBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>边缘检测</string>
   </property>
  </widget>
  <widget class="QLabel" name="conImgLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="trackImgLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>390</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="conImgBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>610</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>轮廓提取</string>
   </property>
  </widget>
  <widget class="QPushButton" name="trackImgBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>610</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>轮廓跟踪</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
    <item>
     <widget class="QComboBox" name="edgeComBox">
      <property name="currentText">
       <string>Canny算子</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Canny算子</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sobel算子</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Laplacian算子</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
    <item>
     <widget class="QLabel" name="edgeLabel">
      <property name="font">
       <font>
        <family>Adobe Arabic</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>边缘检测算子</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
